### ISSUE TITLE
Regression, false negative E302

### ISSUE THREAD

**Issue by asottile** [PyCQA/pycodestyle#858]
### sample code

```python
def f():
    pass

# wat
@hi
def g():
    pass
```

### 2.5.0 behaviour

```console
$ ./venv/bin/pycodestyle --version
2.5.0
$ ./venv/bin/pycodestyle  t.py
$ echo $?
0
```

### 2.4.0 behaviour

```console
$ ./venv/bin/pycodestyle --version
2.4.0
$ ./venv/bin/pycodestyle  t.py 
t.py:5:1: E302 expected 2 blank lines, found 1
```

### git bisection

```console
$ git bisect start
$ git bisect good 2.4.0
$ git bisect bad 2.5.0
$ git bisect run bash -c '! python3 pycodestyle.py ../t.py'
...
running bash -c ! python3 pycodestyle.py ../t.py
0138bb106403241564a0c5af67886c05a2f4f690 is the first bad commit
commit 0138bb106403241564a0c5af67886c05a2f4f690
Author: Antony Lee <anntzer.lee@gmail.com>
Date:   Wed Nov 28 00:43:17 2018 +0100

    Allow omitting blank lines around one-liner definitions.

:100755 100755 c9e1667150d1a78fd236f8272906f3862425b3d5 27009c648ff090847cf78ec26c83fbd79be11d5a M	pycodestyle.py
:040000 040000 c138e2e23b06e384dfc052ffdfaaf372d203c480 817cc37bedea6034c3cf06468c42811b0111de27 M	testsuite
bisect run success
```

0138bb106403241564a0c5af67886c05a2f4f690

CC @anntzer
